### PR TITLE
Correct primary key column output for show create table

### PR DESCRIPTION
**Pull request by zachmu**
Used the column order before, instead of the key order

### CODE REVIEW

File: sql/core.go [hunk 1]
```diff
@@ -308,6 +308,12 @@ type SchemaTarget interface {
 	WithTargetSchema(Schema) (Node, error)
 }
 
+// PrimaryKeySchemaTarget is a node that has a primary key target schema that can be set
+type PrimaryKeySchemaTarget interface {
+	SchemaTarget
+	WithPrimaryKeySchema(schema PrimaryKeySchema) (Node, error)
```
[the remark this answers is not in the document]
Comment (zachmu) [reply sql/core.go:314]: It's not duplicated: this won't work on tables without a primary key if I drop SchemaTarget

**Review by max-hoffman — APPROVED** [review]
lgtm

File: sql/core.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (max-hoffman) [reply sql/core.go:314]: do we have a test for the keyless table case?

**Review by reltuk — APPROVED** [review]
LGTM!

File: sql/core.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (zachmu) [reply sql/core.go:314]: We did not, I added one

**Pull request merged**